src/components/Calculator.js

```diff
@@ -1,32 +1,50 @@
+
+    return (show || '0');
+  };
+  
   render() {
     return (
       <div className="calculator">
         <input type="number" />
         <section className="buttons">
-          <button type="button">AC</button>
-          <button type="button">+/-</button>
-          <button type="button">%</button>
-          <button type="button" className="orange">&#247;</button>
-          <button type="button">7</button>
-          <button type="button">8</button>
-          <button type="button">9</button>
-          <button type="button" className="orange">x</button>
-          <button type="button">4</button>
-          <button type="button">5</button>
-          <button type="button">6</button>
+          <button type="button" onClick={(ev) => this.handleClick(ev.target.name)}>AC</button>
+          <button type="button" onClick={(ev) => this.handleClick(ev.target.name)}>+/-</button>
+          <button type="button" onClick={(ev) => this.handleClick(ev.target.name)}>%</button>
+          <button type="button" className="orange" onClick={(ev) => this.handleClick(ev.target.name)}>&#247;</button>
+          <button type="button" onClick={(ev) => this.handleClick(ev.target.name)}>7</button>
+          <button type="button" onClick={(ev) => this.handleClick(ev.target.name)}>8</button>
+          <button type="button" onClick={(ev) => this.handleClick(ev.target.name)}>9</button>
+          <button type="button" className="orange" onClick={(ev) => this.handleClick(ev.target.name)}>x</button>
+          <button type="button" onClick={(ev) => this.handleClick(ev.target.name)}>4</button>
+          <button type="button" onClick={(ev) => this.handleClick(ev.target.name)}>5</button>
+          <button type="button" onClick={(ev) => this.handleClick(ev.target.name)}>6</button>
           <button type="button" className="orange">-</button>
-          <button type="button">1</button>
-          <button type="button">2</button>
-          <button type="button">3</button>
-          <button type="button" className="orange">+</button>
-          <button type="button" className="zero">0</button>
-          <button type="button">.</button>
-          <button type="button" className="orange">=</button>
+          <button type="button" onClick={(ev) => this.handleClick(ev.target.name)}>1</button>
+          <button type="button" onClick={(ev) => this.handleClick(ev.target.name)}>2</button>
+          <button type="button" onClick={(ev) => this.handleClick(ev.target.name)}>3</button>
+          <button type="button" className="orange" onClick={(ev) => this.handleClick(ev.target.name)}>+</button>
+          <button type="button" className="zero" onClick={(ev) => this.handleClick(ev.target.name)}>0</button>
+          <button type="button" onClick={(ev) => this.handleClick(ev.target.name)}>.</button>
+          <button type="button" className="orange" onClick={(ev) => this.handleClick(ev.target.name)}>=</button>
         </section>
       </div>
     );
```

Comment:
- [ ] Please fix your `click event` currently am not able to do anything with your calculator and this project was about working on the events

Reply:
Done